### DISCUSSION TITLE
Start WebXR Via Button Either Outside the Viewer or With No 3D Viewer

### DISCUSSION THREAD

**Discussion by allenwixted** [google/model-viewer#2598] [Q&A]
Hi there, 

Is it possible to start the AR session in WebXR and Quicklook via a button that's outside the viewer itself? Or via just a button alone?

<img width="287" alt="Screen Shot 2021-07-21 at 09 33 06" src="https://user-images.githubusercontent.com/9553402/126459168-6baaeebe-df73-4f04-ba33-c5493a9ec123.png">

Something similar to above? 

Thanks
Allen

**Comment by elalish** [comment]
Yes, look at #1954.

**Reply by allenwixted** [reply]
Thanks Elalish! Got this working via a second instance in the button only mode. So we've a 3D only mode and an AR only mode
